src/olympia/devhub/templates/devhub/includes/addon_details.html

```diff
@@ -55,7 +55,9 @@
 {% if addon.has_listed_versions() %}
     {% if addon.has_unlisted_versions() %}
       <li class="distribution-tag-listed">
-        <span class="distribution-tag-listed">AMO</span>
+        <span class="distribution-tag-listed tooltip"
+              title="{{ _('Listed on this site after passing code review. Automatic updates are handled by this site.') }}"
```

Comment:
heh, you've got a point, but: https://github.com/mozilla/addons-server/issues/4398#issuecomment-274137542